tests/.suitespec.json

```diff
@@ -242,7 +242,8 @@
             "ddtrace/contrib/asynctest/*"
         ],
         "logging": [
-            "ddtrace/contrib/logging/*"
+            "ddtrace/contrib/logging/*",
+            "tests/contrib/logging/*"
```

Comment:
Maybe best to have this in the `test_logging` suite instead of in this `component` so that we don't run other suites when we touch the test sources.